namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/ecr.tf

```diff
@@ -5,27 +5,12 @@
  *
  */
 module "ecr_credentials" {
-  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.7"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.7"
+
   team_name = var.team_name
   repo_name = "${var.namespace}-ecr"
 
-  /*
-    By default scan_on_push is set to true. When this is enabled then all images pushed to the repo are scanned for any security
-    / software vulnerabilities in your image and the results can be viewed in the console. For further details, please see:
-    https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html
-    To disable 'scan_on_push', set it to false as below:
-  scan_on_push = "false"
-  */
-
-  # Uncomment and provide repository names to create github actions secrets
-  # containing the ECR name, AWS access key, and AWS secret key, for use in
-  # github actions CI/CD pipelines
-  # github_repositories = ["my-repo"]
-
-  # list of github environments, to create the ECR secrets as environment secrets
-  # https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets
-  # github_environments = ["my-environment"]
-
+  github_repositories = [var.namespace]
```

Comment:
Are your namespace and github name the same

Reply:
Yep, both `approved-premises-prototype`